.gitignore

```diff
@@ -7,3 +7,4 @@ dist
 .vscode
 .idea
 .editorconfig
+/scrapers
```

Comment:
This is not standard, it's a local directory - so maybe we should not have it here.